test/compilable/cppmangle.d

```diff
@@ -364,13 +364,7 @@ extern(C++)
     }
 }
 
-version (OSX)
-{
-    static assert(T.foo.mangleof == "__ZNK1T3fooEi");
-    static assert(T.bar.mangleof == "__ZN1T3barEi");
-    static assert(T.boo.mangleof == "__ZN1T3booE");
-}
```

Comment:
What's the motivation for removing the OSX test completely?

Reply:
Matches gdc and ldc behaviour.

Reply:
Ah sorry. I just saw the removed test, not that it will still be executed.